httpcore5/src/main/java/org/apache/hc/core5/net/InetAddressUtils.java

```diff
@@ -137,6 +137,18 @@ public static boolean isIPv6Address(final String input) {
         return isIPv6StdAddress(input) || isIPv6HexCompressedAddress(input);
     }
 
+    /**
+     * Checks whether the parameter is a valid rfc2732 url formatted bracketed IPv6 address (including compressed).
```

Comment:
URL is an acronym and should be in caps.

Reply:
Good call, updated.

Reply:
@carterkozak I would rather avoid referencing RFCs in class names and in javadocs at all. RFCs can get superseded by a newer version and one would have to scan the source for references to outdated documents.

Reply:
That's fair, I've removed references to the rfc with the exception of a line-comment in the test.